.github/workflows/translations.yaml

```
@@ -17,6 +17,7 @@ env:
 jobs:
   upload:
     name: Upload
+    if: secrets.LOKALISE_TOKEN != null
```

I don't think the `secrets` context is available in these statements here (the context supported by `if` in jobs is limited).

---

Hmm, ok, thanks. I need to go back and check the docs more thoroughly. Or do you have any good suggestions off hand?

---

You could make an extra step that checks it (e.g. check in shell if it exists and set a variable that can be check in this step).

Or, I think we can simply lock this down to the Home Assistant org.

```suggestion
    if: github.repository_owner == 'home-assistant'
```

---

If that's fine by you, even simpler and better 👍